spec/features/signup_spec.rb

```diff
@@ -0,0 +1,13 @@
+RSpec.feature 'Signup' do 
+ 
+  scenario 'user can signup' do 
+    visit('/signup')
+    fill_in :firstname, with: 'Carol'
+    fill_in :surname, with: 'Gardiner'
+    fill_in :username, with: 'cazzy'
+    fill_in :email, with: 'caz@caz.com'
+    fill_in :password, with: 'secret123'
+    click_on 'Sign up'
+    expect(page.current_path).to eq '/profile/6'
+  end
+end
```

Comment:
Your feature test's names and flow make what you are testing super clear, and as a product of that I know what functionality your app should have which is great. If you had time it would be good to add more of these to show some of your apps other functionalities, such as the requirement for emails/username's to be unique on sign up.